bool cmd
bool topic
---
int16[] vecCen